core/src/test/scala/org/genivi/sota/core/daemon/TreehubCommitListenerSpec.scala

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright: Copyright (C) 2016, Jaguar Land Rover
+ * License: MPL-2.0
+ */
+package org.genivi.sota.core.daemon
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import com.advancedtelematic.libats.data.RefinedUtils.RefineTry
+import java.security.MessageDigest
+import java.util.UUID
+import org.genivi.sota.DefaultPatience
+import org.genivi.sota.core._
+import org.genivi.sota.core.client.FakeReposerverClient
+import org.genivi.sota.core.data.Package
+import org.genivi.sota.core.db.Packages
+import org.genivi.sota.core.resolver.DefaultConnectivity
+import org.genivi.sota.core.transfer.DefaultUpdateNotifier
+import org.genivi.sota.data._
+import org.genivi.sota.messaging.LocalMessageBus
+import org.genivi.sota.messaging.Messages.TreehubCommit
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSuite, ShouldMatchers}
+import scala.concurrent.ExecutionContext
+
+class TreehubCommitListenerSpec extends FunSuite
+  with DatabaseSpec
+  with ShouldMatchers
+  with ScalaFutures
+  with DefaultPatience
+  with Namespaces
+  {
+
+  implicit val ec = ExecutionContext.global
+  implicit val system = ActorSystem("TreehubCommitListenerSpec")
```

Comment:
Could we have a `system.terminate` when this test is done?
Add a `with org.scalatest.BeforeAndAfterAll`
```
override def afterAll(): Unit = {
  super.afterAll()
  system.terminate()
}
```